server/src/main/java/org/apache/cloudstack/acl/RoleManagerImpl.java

```diff
@@ -146,10 +150,89 @@ public RoleVO doInTransaction(TransactionStatus status) {
         });
     }
 
+    @Override
+    @ActionEvent(eventType = EventTypes.EVENT_ROLE_CREATE, eventDescription = "creating Role")
```

Comment:
Should the description say we're creating a role by cloning another role ?

Reply:
> Should the description say we're creating a role by cloning another role ?

Updated event description